Makefile

```diff
@@ -82,7 +85,7 @@ tailscale-debug: $(DEBUG_APK) ## Build the debug APK
 
 .PHONY: release
 release: tailscale.jks $(RELEASE_AAB) ## Build the release AAB
-	jarsigner -sigalg SHA256withRSA -digestalg SHA-256 -keystore tailscale.jks $(RELEASE_AAB) tailscale
+	jarsigner -sigalg SHA256withRSA -digestalg SHA-256 -keystore tailscale.jks -storepass $(JKS_PASSWORD) $(RELEASE_AAB) tailscale
```

Comment:
does this fail if i make a release build but I don't have a password?

Reply:
The release build always required that you enter JKS password to enter manually.  There will be  a check in the corp builder to ensure JKS_PASSWORD is set and we'll error out before we try the build (and defer the remove-image job).  I'll also add a post build check to make sure things are properly signed.